sdk/cosmos/azure-cosmos-encryption/pom.xml

```diff
@@ -203,7 +203,7 @@ Licensed under the MIT License.
                 <include>com.fasterxml.jackson.datatype:jackson-datatype-jsr310:[2.10.1]</include> <!-- {x-include-update;com.fasterxml.jackson.datatype:jackson-datatype-jsr310;external_dependency} -->
                 <include>com.fasterxml.jackson.module:jackson-module-afterburner:[2.10.1]</include> <!-- {x-include-update;cosmos_com.fasterxml.jackson.module:jackson-module-afterburner;external_dependency} -->
                 <include>io.dropwizard.metrics:metrics-core:[4.1.0]</include> <!-- {x-include-update;cosmos_io.dropwizard.metrics:metrics-core;external_dependency} -->
-                <include>io.micrometer:micrometer-core:[1.2.0]</include> <!-- {x-include-update;io.micrometer:micrometer-core;external_dependency} -->
+                <include>io.micrometer:micrometer-core:[1.3.0]</include> <!-- {x-include-update;io.micrometer:micrometer-core;external_dependency} -->
```

Comment:
Please remove this line instead of updating it, this isn't a dependency of this library and Only non-test dependencies need entries here. @kushagraThapar it looks like this section was copied/pasted from benchmark. There are other includes listed here that aren't actual library dependencies and should be removed.

Reply:
Good catch @JimSuplizio .. @chenrujun - please remove the above line from azure-cosmos-encryption pom.xml
Also, I agree with @JimSuplizio  here - we should also update `io.micrometer:micrometer-registry-graphite` to 1.3.0
Please do so.

Reply:
Done.